### ISSUE TITLE
PMMH tests fail

### ISSUE THREAD

**Issue by trappmartin** [TuringLang/Turing.jl#463]
The `PMMH` tests currently fail on the master branch. 

```bash
Test Failed
Expression: ≈(mean(chain[:s]), 49 / 24, atol=0.2)  
ERROR: LoadError: There was an error during testing
while loading /Users/martin/.julia/v0.6/Turing/test/pmmh.jl/pmmh.jl, in expression starting on line 22
```

@emilemathieu do you have an idea why that is happening? Is this related to #369 ?

Thanks!
Martin

**Comment by emilemathieu** [comment]
Hi !
I do remember running in weird issues once merged in master, whereas things were working fine on my branch...
Does increasing the number of iterations/particles solve the issue ? 

Cheers,
Emile

**Comment by trappmartin** [comment]
Unfortunately, not. In fact, the difference between the expected value and the computed estimate increasing differs if I increase the number of iterations. See below. Actually, when looking at the test at hand, it seems to me that the test is maybe missing a `sqrt`? Also, 10^3 iterations seems a lot for the problem to me?!?

See below:

```julia
...
alg = PMMH(10^3, SMC(20, :m), MH(1,(:s, (x) -> Normal(x, 1) )))
chain = sample(pmmhtest(x), alg)
info("mean(chain[:s]) = ", mean(chain[:s]))
info("expected value = ", 49/24 )
@test mean(chain[:s]) ≈ 49/24 atol=0.2
```
results in:
```bash
INFO: mean(chain[:s]) = 1.6219249820170678
INFO: expected value = 2.0416666666666665 
```
, and
```julia
alg = PMMH(10^4, ...)
```
results in:
```bash
INFO: mean(chain[:s]) = 1.4284608145719573
```
, and finally
```julia
alg = PMMH(10^5, ...)
```
results in:
```bash
INFO: mean(chain[:s]) = 1.4197014524140301
```
.

**Comment by yebai** [comment]
Fixed by #650
